cmd/vic-machine/configure/configure.go

```diff
@@ -0,0 +1,245 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configure
+
+import (
+	"context"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"gopkg.in/urfave/cli.v1"
+
+	"github.com/vmware/vic/cmd/vic-machine/common"
+	"github.com/vmware/vic/lib/config"
+	"github.com/vmware/vic/lib/install/data"
+	"github.com/vmware/vic/lib/install/management"
+	"github.com/vmware/vic/lib/install/validate"
+	"github.com/vmware/vic/pkg/errors"
+	"github.com/vmware/vic/pkg/trace"
+	"github.com/vmware/vic/pkg/vsphere/vm"
+)
+
+// Configure has all input parameters for vic-machine configure command
+type Configure struct {
+	*data.Data
+
+	upgrade  bool
+	executor *management.Dispatcher
+}
+
+func NewConfigure() *Configure {
+	configure := &Configure{}
+	configure.Data = data.NewData()
+
+	return configure
+}
+
+// Flags return all cli flags for configure
+func (c *Configure) Flags() []cli.Flag {
+	util := []cli.Flag{
+		cli.BoolFlag{
+			Name:        "force, f",
+			Usage:       "Force the configure operation",
+			Destination: &c.Force,
+		},
+		cli.DurationFlag{
+			Name:        "timeout",
+			Value:       3 * time.Minute,
+			Usage:       "Time to wait for configure",
+			Destination: &c.Timeout,
+		},
+		cli.BoolFlag{
+			Name:        "resetInProgressFlag",
```

Comment:
Can this be changed to `reset-progress` or something since none of the other flags are camel case?